src/entity2input.js

```diff
@@ -1,19 +1,21 @@
 const { camelCase, upperFirst } = require('lodash')
-const { entityField2gql } = require("./helpers/gqlConverters")
+const { entityField2gql, EnumConventions } = require("./helpers/gqlConverters")
 const { checker } = require('suma')
 const { entityValidator } = require('./herbsValidator')
 
-function entity2input(entity, isCamelCase = true, customName ) {
+function entity2input(entity, options = undefined) {
   const validation = entityValidator(entity)
   if (!checker.isEmpty(validation)) {
     const error = Error('InvalidEntity')
     error.invalidArgs = validation
     throw error
   }
 
+  const convention  = EnumConventions[options?.convention]
+
   let name
-  if (customName) name = customName
-  else if (!isCamelCase) name = entity.name
+  if (options?.customName) name = options.customName
+  else if (convention) name = convention(entity.name)
```

Comment:
`else if (convention.entityName) name = convention.entityName(entity.name)`
the idea is that it could have many other conventions other than entity names

Reply:
solved